Engine/Source/Globals.h

```diff
@@ -1,9 +1,12 @@
+#include <string>
+using namespace std;
```

Comment:
I think we should about using namespaces for now. For readability, if there are not a lot of noise, in my opinion I think that is better to see from which package a function or data belongs to.